routes/gamesapi.js

```diff
@@ -4,16 +4,31 @@ const router = express.Router();
 const mdb = require('./dbapi');
 const db = new mdb.Database();
 const config = require('./config');
+const Cryptr = require('cryptr');
 
 router.get('/', function(req, res, next) {
   res.send('Games server is running.');
 });
 
-// curl http://localhost:3000/gamesapi/randomwords
+/*
+curl -H "Cookie: userid=588d937f009f04028792504e" \
+http://localhost:3000/gamesapi/randomwords
+*/
 router.get('/randomwords', function(req, res, next) {
   db.getRandomWords(config.WORD_COUNT, (words) => {
-      if (words) res.send(JSON.stringify({'words': words}));
-      else res.status(500).send('Error occurred.')
+    if (!words) return res.status(500).send('Error occurred.');
+    if (config.SEND_ENCRYPTED_WORD) {
+      if (!req.headers.hasOwnProperty('cookie')) {
+        return res.status(400).send('Cookie not provided.');
+      }
+      const sessionId = req.headers['cookie'].substr(config.COOKIE_NAME.length + 1); // remove "cookieName="
```

Comment:
my bad i forgot to add cookie parser
https://github.com/expressjs/cookie-parser
can you please add this and use this?
will make everyone's life easier